src/boot/measure.c

```diff
@@ -195,7 +195,7 @@ static int parse_argv(int argc, char *argv[]) {
         assert(argv);
 
         /* Make sure the arguments list and the section list, stays in sync */
-        assert_cc(_ARG_SECTION_FIRST + _UNIFIED_SECTION_MAX == _ARG_SECTION_LAST + 1);
+        // assert_cc(_ARG_SECTION_FIRST + _UNIFIED_SECTION_MAX == _ARG_SECTION_LAST + 1);
```

Comment:
please add a comment about the temporary thing.

Reply:
ah, it's the immediately following commit that drops this again, that should be explanatory enough.